test/integration/integration_config.py

```diff
@@ -0,0 +1,24 @@
+import os
+import yaml
+
+
+class IntegrationConfig:
+    def __init__(self):
+        dir_path = os.path.dirname(os.path.realpath(__file__))
+        path = os.path.join(dir_path, "clarity_steps.yml")
```

Comment:
So we have a rather inconvenient situation that we must do some integration tests against the LIMS server. Up until now, we've been hardcoding values, but now one can access them from a configuration file, which describes the data that needs to be in place on the server. This makes it easier for us to change it later or for external parties to set up the LIMS in such a state that they can validate it.

I use it in a later commit, here: 5da62ce, test/integration/service/test_artifact_service.py